kalite/store/views.py

```diff
@@ -10,9 +10,4 @@
 from django.shortcuts import get_object_or_404
 from django.utils.translation import ugettext as _
 
-from kalite.shared.decorators import require_login
```

Comment:
Can't delete the complete store app, As we need to keep the Store Models such as to be in sync with central server, Right?